about.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>Web Conference 2021</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@0,400;0,500;0,600;0,700;0,800;0,900;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css"
+      type="text/css"
+    />
+    <link href="styles.css" rel="stylesheet" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="first-nav">
+        <i class="fa fa-times icon cursor xl-none none" id="close"></i>
+        <i class="fa fa-bars icon cursor xl-none" id="open"></i>
+          <ul class="first-nav-ul">
+              <li class="first-nav-list"><a href="#" class="first-nav-anchor"><i class="fa fa-facebook-f"></i></a></li>
+              <li class="first-nav-list"><a href="#" class="first-nav-anchor"><i class="fa fa-twitter"></i></a></li>
+              <li class="first-nav-list"><a href="#" class="first-nav-anchor">English</a></li>
+              <li class="first-nav-list"><a href="#" class="first-nav-anchor">My page</a></li>
+              <li class="first-nav-list"><a href="#" class="first-nav-anchor">Logout</a></li>
+          </ul>
+          </nav>
+      <nav class="second-nav none" id="nav">
+          <ul class="second-nav-logo">
```

Comment:
- [ ] On the design template, the logo has some left margin. You should try to make your project look like the template as much as possible.

Original Design:
![Screenshot from 2021-07-01 19-59-43](https://user-images.githubusercontent.com/1642014/124162176-e7019900-daa6-11eb-9dd8-283464fd7228.png)

Reply:
:+1: Addressed